types/v56/types.go

```diff
@@ -257,13 +257,13 @@ type NetworkConfigSection struct {
 // Since: 0.9
 type NetworkConnection struct {
 	Network                 string `xml:"network,attr"`                      // Name of the network to which this NIC is connected.
-	NetworkConnectionIndex  int    `xml:"NetworkConnectionIndex"`            // Virtual slot number associated with this NIC. First slot number is 0.
 	NeedsCustomization      bool   `xml:"needsCustomization,attr,omitempty"` // True if this NIC needs customization.
-	ExternalIPAddress       string `xml:"ExternalIpAddress,omitempty"`       // If the network to which this NIC connects provides NAT services, the external address assigned to this NIC appears here.
```

Comment:
Why this has been removed?

Reply:
It isn't, I shuffled to be inline with strict xml cause otherwise serialized messages isn't accepted by vCD.